### PR TITLE
Feat: add vela minimal chart

### LINKED ISSUE #2265: [Feature] add a minimalist version kubevela chart
We can name it `vela-minimal`, it works for minimal requirements.

## Usage

* Works as only one controller process, require minimal resources, for one cluster only. 
* Compose Kubernetes resources with a simplified way.
* Highly extensibility that can glue any resources that already available in the cluster, don't install any addons for users. Users can self service for any needed features and register into KubeVela.

## All Necessary CRDs and features

CRD Name | Usage
------------ | -------------
applications.core.oam.dev  | KubeVela Core Application Object
applicationrevisions.core.oam.dev | The revision CRD of Application
componentdefinitions.core.oam.dev | ComponentDefinition Object
traitdefinitions.core.oam.dev  | TraitDefinition Object
workflowstepdefinitions.core.oam.dev | Workflowstep Object
scopedefinitions.core.oam.dev | ScopeDefinition Object
policydefinitions.core.oam.dev | PolicyDefinition Object
 workloaddefinitions.core.oam.dev  | WorkloadDefinition Object
definitionrevisions.core.oam.dev | The revision CRD for all definition objects
rollouts.standard.oam.dev | Rollout feature trait
resourcetrackers.core.oam.dev | Garbage Collection feature
healthscopes.core.oam.dev | Health Check feature

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:
add vela minimal chart

**Which issue(s) this PR fixes**:

Fixes #2265

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/workflows/registry.yml [hunk 1]
```diff
@@ -145,6 +148,7 @@ jobs:
       - name: add artifacthub stuff to the repo
         run: |
           rsync README.md $HELM_CHART/README.md
+          rsync README.md $MINIMAL_HELM_CHART/README.md
```
Comment (wonderflow) [inline .github/workflows/registry.yml:151]: 单独写一个readme，直接放到 minimal的chart里

File: charts/vela-minimal/values.yaml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+apiServer:
+  enabled: true
+  port: 8000
+  replicaCount: 1
+  Service:
+    type: ClusterIP
+
+multicluster:
+  enabled: false
```
Comment (wonderflow) [inline charts/vela-minimal/values.yaml:114]: does it really work if we enable here?

File: charts/vela-minimal/values.yaml [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+concurrentReconciles: 4
+
+# dependCheckWait is the time to wait for ApplicationConfiguration's dependent-resource ready
+dependCheckWait: 30s
+
+# OAMSpecVer is the oam spec version controller want to setup
+OAMSpecVer: "minimal"
+
+apiServer:
+  enabled: true
```
Comment (wonderflow) [inline charts/vela-minimal/values.yaml:107]: does it work?

File: charts/vela-minimal/values.yaml (hunk 2 above)
Comment (FogDong) [reply charts/vela-minimal/values.yaml:114]: The multi-cluster relies on env binding and will be verified later after env binding is part of workflow.

**Review by wonderflow — APPROVED** [review]

**Review by yangsoon — APPROVED** [review]
/lgtm

**Pull request merged**